datafusion/core/tests/tpcds_planning.rs

```diff
@@ -320,6 +320,7 @@ async fn tpcds_logical_q57() -> Result<()> {
 }
 
 #[tokio::test]
+#[ignore] // decorrelate disjunction decorrelate subquery isn't supported.
```

Comment:
is the issue that when we turn on "skip failed optimizer rules" this query errors?

Reply:
I recommend putting a link to https://github.com/apache/arrow-datafusion/issues/5368 in the comment as well so this is easier to find / reneable

Reply:
> I recommend putting a link to #5368 in the comment as well so this is easier to find / reneable

Great idea.